libvast/src/system/importer.cpp

```diff
@@ -172,10 +182,17 @@ behavior importer(stateful_actor<importer_state>* self, path dir,
       VAST_DEBUG(self, "registers index", index);
       self->send(self->state.index, sys_atom::value, put_atom::value, index);
     },
+    // TODO: some atom ?
```

Comment:
Only if IMPORTER receives more than a single actor in a message. Until then, less is more :-).